### DISCUSSION TITLE
Question - Where to get help

### DISCUSSION THREAD

**Discussion by AstroVinny** [sly9/VoyagerTelegramBot#43] [Q&A] [answered]
Is there a place to get help with VoyagerTelegramBot?  I have it installed and have it connecting to my bot.  I get a message that says:

> Connected to <mycomputername> (127.0.0.1) [Release 2.3.5f Build 2022-02-09]

from my Telegram App on my phone, so I know it is talking to Telegram properly.

However, the screen says that the Main Camera and Mount say "Disconnected" and they are connected.  I get no other messages through the interface.

Thank you in advance for your assistance.
Tim.

**Comment by bigpizza** [comment]
Hi Tim,

Thank you for contacting us. When starting VoyagerTelegramBot, the status of camera and mount may show "disconected" until it receives update message from Voyager. This process may take several seconds.

If you still have such issue when devices are connected to Voyager but shown as disconnected in VoyagerTelegramBot for over a minutes, I would like to ask you to help us debug our app (we majorly tested our tool  only with Paramount and ZWO cameras).

To do so, please:
1. Open the "config.yml" file with text file editor and change "should_dump_log: **False**" to "should_dump_log: **True**". This change will keep every messages we receive from Voyager to a single file.
2. Keep the VoyagerTelegramBot open and do your regular shoot sessions.
3. After 10min or 2 images were taken, you could close the VoyagerTelegramBot, the log file should be saved in the same folder.
4. Since the log file contains some personal identifier such as name and computer name, I recommend you to remove them or send it to "mu(dot)astro(dot)4512(at)gmail".

Please let me know if you have any questions :D

Regards,

Kun

**Comment by AstroVinny** [comment]
Kun:

Thank you for your reply.  I really appreciate your time.

I started the application with the should_dump_log set to True.  I
immediately noticed this entry in the log file:

{"Event":"Version","Timestamp":1648127693.53115,"Host":"DESKTOP-1BHRO2T","Inst":1,"VOYVersion":"Release
2.3.5f - Built 2022-02-09","VOYSubver":"","MsgVersion":1}
{"jsonrpc": "2.0", "error": {"code": 1, "message": "Authentication Level
not Allow this request"}, "id": 1}

The second line didn't look good to me.  I thought I would check with you.
I will leave it running and send the entire log file to you in the morning
if I don't hear back.

Best.
Tim.

On Thu, Mar 24, 2022 at 2:11 AM Kun Wang ***@***.***> wrote:

> Hi Tim,
>
> Thank you for contacting us. When starting VoyagerTelegramBot, the status
> of camera and mount may show "disconected" until it receives update message
> from Voyager. This process may take several seconds.
>
> If you still have such issue when devices are connected to Voyager but
> shown as disconnected in VoyagerTelegramBot for over a minutes, I would
> like to ask you to help us debug our app (we majorly tested our tool only
> with Paramount and ZWO cameras).
>
> To do so, please:
>
>    1. Open the "config.yml" file with text file editor and change
>    "should_dump_log: *False*" to "should_dump_log: *True*". This change
>    will keep every messages we receive from Voyager to a single file.
>    2. Keep the VoyagerTelegramBot open and do your regular shoot sessions.
>    3. After 10min or 2 images were taken, you could close the
>    VoyagerTelegramBot, the log file should be saved in the same folder.
>    4. Since the log file contains some personal identifier such as name
>    and computer name, I recommend you to remove them or send it to
>    "mu(dot)astro(dot)4512(at)gmail".
>
> Please let me know if you have any questions :D
>
> Regards,
>
> Kun
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/sly9/VoyagerTelegramBot/discussions/43#discussioncomment-2427700>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARSMABBIJFGX6FAFKE5YMTDVBQBSTANCNFSM5RLJBDZA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Tim Hutchison
***@***.***

**Comment by AstroVinny** [comment]
Some further info.  This is what I get in Voyager's Application Server
window when VoyagerBot starts:

[image: image.png]

FYI - The Voyager Web Dashboard works just fine.  I believe you are using
the same interface...

Thank you again for any help you can provide.
Tim.

On Thu, Mar 24, 2022 at 2:11 AM Kun Wang ***@***.***> wrote:

> Hi Tim,
>
> Thank you for contacting us. When starting VoyagerTelegramBot, the status
> of camera and mount may show "disconected" until it receives update message
> from Voyager. This process may take several seconds.
>
> If you still have such issue when devices are connected to Voyager but
> shown as disconnected in VoyagerTelegramBot for over a minutes, I would
> like to ask you to help us debug our app (we majorly tested our tool only
> with Paramount and ZWO cameras).
>
> To do so, please:
>
>    1. Open the "config.yml" file with text file editor and change
>    "should_dump_log: *False*" to "should_dump_log: *True*". This change
>    will keep every messages we receive from Voyager to a single file.
>    2. Keep the VoyagerTelegramBot open and do your regular shoot sessions.
>    3. After 10min or 2 images were taken, you could close the
>    VoyagerTelegramBot, the log file should be saved in the same folder.
>    4. Since the log file contains some personal identifier such as name
>    and computer name, I recommend you to remove them or send it to
>    "mu(dot)astro(dot)4512(at)gmail".
>
> Please let me know if you have any questions :D
>
> Regards,
>
> Kun
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/sly9/VoyagerTelegramBot/discussions/43#discussioncomment-2427700>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARSMABBIJFGX6FAFKE5YMTDVBQBSTANCNFSM5RLJBDZA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Tim Hutchison
***@***.***

**Comment by AstroVinny** [comment, marked as answer]
Kun:

I'm so sorry for the multiple emails.  The issue seems to be that the
Application Server Authentication was set to "None" in Voyager.  When I set
it to "User Password" and set the user name and password in VoyagerBot it
started working.

Thank you for your help.
Tim.

On Thu, Mar 24, 2022 at 2:11 AM Kun Wang ***@***.***> wrote:

> Hi Tim,
>
> Thank you for contacting us. When starting VoyagerTelegramBot, the status
> of camera and mount may show "disconected" until it receives update message
> from Voyager. This process may take several seconds.
>
> If you still have such issue when devices are connected to Voyager but
> shown as disconnected in VoyagerTelegramBot for over a minutes, I would
> like to ask you to help us debug our app (we majorly tested our tool only
> with Paramount and ZWO cameras).
>
> To do so, please:
>
>    1. Open the "config.yml" file with text file editor and change
>    "should_dump_log: *False*" to "should_dump_log: *True*". This change
>    will keep every messages we receive from Voyager to a single file.
>    2. Keep the VoyagerTelegramBot open and do your regular shoot sessions.
>    3. After 10min or 2 images were taken, you could close the
>    VoyagerTelegramBot, the log file should be saved in the same folder.
>    4. Since the log file contains some personal identifier such as name
>    and computer name, I recommend you to remove them or send it to
>    "mu(dot)astro(dot)4512(at)gmail".
>
> Please let me know if you have any questions :D
>
> Regards,
>
> Kun
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/sly9/VoyagerTelegramBot/discussions/43#discussioncomment-2427700>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARSMABBIJFGX6FAFKE5YMTDVBQBSTANCNFSM5RLJBDZA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Tim Hutchison
***@***.***

**Reply by bigpizza** [reply]
Glad to hear that you got it work!
